morpheus/app/views.py

```diff
@@ -133,17 +138,8 @@ class SmsBillingView(ServiceView):
     async def call(self, request) -> PreResponse:
         m = await self.request_data(SmsBillingModel)
         company_code = self.request.match_info['company_code']
-        total_spend = await self.app['pg'].fetchval(
-            """
-            select sum(m.cost)
-            from messages as m
-            join message_groups j on m.group_id = j.id
-            where j.company=$1 and send_ts between $2 and $3
-            """,
-            company_code,
-            m.start,
-            m.end,
-        )
+        method = self.request.match_info['method']
```

Comment:
better to hardcode method. If someone enters something funky in here, it'll throw a DB error.

Reply:
or set method properly in the URL, but I think that's unnecessary.